cypress/selectorHelpers/index.jsx

```diff
@@ -0,0 +1,24 @@
+const selectDataHook = hook => `[data-test="${hook}"]`; // writes to component
+const writeDataHook = hook => ({ "data-test": hook }); // writes to selectors file
```

Comment:
This writes to the component cause you're spreading this object onto a React component like``` <Component {...{ data-test: "someElement" }} />```